### DISCUSSION TITLE
App Updates not detected

### DISCUSSION THREAD

**Discussion by jrudlin** [svrooij/WinTuner#63] [General]
Get-WtWin32Apps -Username (whoami /upn)

LatestVersion       :
IsUpdateAvailable   : False
PackageId           :
Name                : Android Studio
CurrentVersion      : **2023.2.1.23**
GraphId             : 6fbc3f06-15ca-4e03-87e7-xxxxxxxxx
SupersededAppCount  : 0
SupersedingAppCount : 0

From:
https://[raw.githubusercontent.com/svrooij/winget-pkgs-index/main/index.json](https://raw.githubusercontent.com/svrooij/winget-pkgs-index/main/index.json)
"Google.AndroidStudio","Version":"2023.3.1.18"

Also, in WinGet Search:
![image](https://github.com/svrooij/WingetIntune/assets/29498781/206f9cae-2681-469c-b733-af9d126f6c9b)

**Comment by svrooij** [comment]
This is the same issue as #64
